### PR TITLE
Pilot loggers

### PR DESCRIPTION
**Pull request by wkrzemien**
Pilot Logging system:
PilotLogger is used in the extended version (ExtendedLogger) of the Logger class, which is used by the pilot commands. The PilotLogger is by default off and the ExtendedLogger acts in exactly the same manner as the original Logger class.
The PilotLogger can send messages about the current status of the installation/configuration etc. to RabbitMQ server from which it will be retrieved by the consumer process and stored into Pilot DB
The messages contain timestamps, status information and the pilotUUID (not pilotID!) by which we can identify even pilots which dont have the pilotID assigned.
In addition, the PilotLogger is storing some of the messages locally just before checking the connection to the RabbitMQ server, if the connection is down, the messages are queued if the connection is back the whole set of messages are sent.
The current version uses STOMP protocol and ssl certificates for authentication

Added getUniqueIdFromOS to set the uuid depending on the OS e.g. CREAM, GRID, VM et

### CODE REVIEW

**Comment by fstagni** [comment]
Looks good to me. I would just make a trial of using python `logging` module instead of just print.

**Comment by wkrzemien** [comment]
i replaced prints by standard logging calls

File: pytest.ini [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #Configuration file for py.test
 [pytest]
 python_files=Test_*.py
-addopts = -rx -v --color=yes --tb=long --ignore=tests --cov
+addopts = -rx -v --color=yes --tb=long --ignore=tests
```
Comment (fstagni) [inline pytest.ini]: what's the problem with coverage?

**Comment by wkrzemien** [comment]
Fixed.

**Pull request merged**